### PR TITLE
[Developer] Add visibility of models to TIKE UI

### PR DESCRIPTION
**Pull request by mcdurdin**
This PR starts to add .model.ts files to the Keyman Developer UI, so that average users can find their way through creating a model from start to finish. Still to do:

- [x] Separate 'model' and 'keyboard type projects
- [x] Template for model projects
- [x] Good icons
- [x] kmcomp compile project files with .model.ts
- [x] Good content in project view for models, plus working links to help and tutorials
- [x] Add .model.ts files to File|New dialog
- [x] Validation of model filenames at creation time.

Postponing the following to 13.0 or later:
- [ ] Visual editor for wordlist models (basic parse of .ts to find sources array?)
- [ ] Creating a package from a model
- [ ] Properly differentiate between .model.ts and .ts in Developer (harder than it might seem due to long-standing assumptions that extension is final .ts only). This will improve maintenance.

### CODE REVIEW

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/160472:id/kvk/index.html)

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/160479:id/kvk/index.html)

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/160669:id/kvk/index.html)

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/160925:id/kvk/index.html)

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/160930:id/kvk/index.html)

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/160940:id/kvk/index.html)

File: windows/src/developer/kmconvert/data/wordlist-lexical-model/model.model_info [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "license": "mit",
+    "languages": [
+        $LANGUAGES_KEYBOARD_INFO
```
[the remark this answers is not in the document]
Comment (mcdurdin) [reply windows/src/developer/kmconvert/data/wordlist-lexical-model/model.model_info]: No; it takes the same template parameter as the keyboard_info file; I'll leave it as is for now :)

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/161091:id/kvk/index.html)

**Pull request merged**